src/components/SchemaTable/index.jsx

```diff
@@ -1,10 +1,59 @@
 import React from 'react';
 import PropTypes from 'prop-types'; // eslint-disable-line no-unused-vars
+import './index.css'; // temporary stylesheet for testing (TODO: refactor to use MUI styles)
+
+// TODO: eliminate after testing
+const exampleSchema = require('../../../schemas/basicDataTypes/string/stringPattern.json');
 
 function SchemaTable() {
+  const leftRows = [];
+  const rightRows = [];
+
+  const renderDefault = (schema) => {
```

Comment:
A comment explain what `renderDefault` may be good here.

Reply:
Added comments!